### PR TITLE
banner for churn customers FFH-242

### PR DESCRIPTION
**Pull request by sonujangra27**
banner

### What do you want to achieve?
- To show a banner if a customer has been churned

### PR Checklist

- [x] I have tested it locally and all functionalities are working fine.
- [x] I have compared it with mocks and all design elements are the same.
- [ ] I have tested it in IE Browser.

### How was the code tested?

- Hit the api and checked for the plan for that application ID
if the plan is showing churn then show the banner

Calling this api - https://github.com/Kommunicate-io/Kommunicate-Server/pull/745

### What new thing you came across while writing this code? 
-

### In case you fixed a bug then please describe the root cause of it? 
-

### Screenshot
-  
image

NOTE: Make sure you're comparing your branch with the correct base branch

### CODE REVIEW

File: webplugin/js/app/mck-app.js [hunk 1]
```diff
@@ -602,7 +603,31 @@ function ApplozicSidebox() {
     //         }
     //     })
     // };
-
+    function checkForChurnCustomer(){
+        let appId=applozic._globals.appId;
+        let uploadUrl = `http://localhost:3999/users/v2/chat/plugin/get-current-plan?appId=${appId}`
```
Comment (prochnost04) [inline webplugin/js/app/mck-app.js:608]: update the url 
the url should be fetched according to the env in which the widget is deployed

Comment (prochnost04) [reply webplugin/js/app/mck-app.js:608]: Template literals are not allowed
concate the strings

Comment (prochnost04) [reply webplugin/js/app/mck-app.js:608]: use var in web sdk 
(for IE support)

File: webplugin/js/app/mck-app.js [hunk 2]
```diff
@@ -602,7 +603,31 @@ function ApplozicSidebox() {
...
+            if(res.churnStatus=="churn"){
+                var churnCust = document.getElementById("km-churn-customer")
+                if(churnCust){
+                    churnCust.classList.add("is-churn-customer")
+                }     
+            }
+          })
+          .catch(error => {
+            console.error(error);
+          });
```
Comment (prochnost04) [inline webplugin/js/app/mck-app.js:629]: use ajax

**Review by prochnost04 — CHANGES_REQUESTED** [review]

**Comment by sonujangra27** [comment]
@prochnost04 removed the call and function - 

using the settings API response to show or hide the banner

did not use the KOmmunciateUI then I will have to call the API again to use the response.

Used Var wherever needed.

File: webplugin/js/app/mck-app.js [hunk 3]
```diff
@@ -377,6 +377,14 @@ function ApplozicSidebox() {
                 );
             };
 
+            //to check if the customer has been churned then show the churn banner
+            if (data.currentActivatedPlan == "churn") {
+                var churnCust = document.getElementById("km-churn-customer")
+                if (churnCust) {
+                    churnCust.classList.add("is-churn-customer")
```
Comment (prochnost04) [inline webplugin/js/app/mck-app.js:384]: you could have use n-vis ?
This will make the code consistent

Comment (sonujangra27) [reply webplugin/js/app/mck-app.js]: done

**Review by prochnost04 — DISMISSED** [review]

**Review by panwaranuj01 — DISMISSED** [review]

**Pull request closed without merging**